typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -142,56 +166,37 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
         });
         break;
       case IsmType.OP_STACK:
-        assert(
-          this.deployer,
-          `HyperlaneDeployer must be set to deploy ${ismType}`,
-        );
-        contract = await this.deployer.deployContractFromFactory(
-          destination,
-          new OPStackIsm__factory(),
-          IsmType.OP_STACK,
-          [config.nativeBridge],
-        );
+        contract = await this.deployer.deployContract(destination, ismType, [
+          config.nativeBridge,
+        ]);
         break;
       case IsmType.PAUSABLE:
-        assert(
-          this.deployer,
-          `HyperlaneDeployer must be set to deploy ${ismType}`,
-        );
-        contract = await this.deployer.deployContractFromFactory(
+        contract = await this.deployer.deployContract(
           destination,
-          new PausableIsm__factory(),
           IsmType.PAUSABLE,
           [config.owner],
         );
-        await this.deployer.transferOwnershipOfContracts(destination, config, {
-          [IsmType.PAUSABLE]: contract,
-        });
         break;
       case IsmType.TRUSTED_RELAYER:
-        assert(
-          this.deployer,
-          `HyperlaneDeployer must be set to deploy ${ismType}`,
-        );
         assert(mailbox, `Mailbox address is required for deploying ${ismType}`);
-        contract = await this.deployer.deployContractFromFactory(
+        contract = await this.deployer.deployContract(
           destination,
-          new TrustedRelayerIsm__factory(),
           IsmType.TRUSTED_RELAYER,
           [mailbox, config.relayer],
         );
         break;
       case IsmType.TEST_ISM:
-        if (!this.deployer) {
-          throw new Error(`HyperlaneDeployer must be set to deploy ${ismType}`);
-        }
-        contract = await this.deployer.deployContractFromFactory(
+        contract = await this.deployer.deployContract(
           destination,
-          new TestIsm__factory(),
           IsmType.TEST_ISM,
           [],
         );
         break;
+      case IsmType.ARB_L2_TO_L1:
+        contract = await this.deployer.deployContract(destination, ismType, [
+          config.bridge,
+        ]);
+        break;
```

Comment:
Just a small nit: I see that in some `.deployContract` calls we use the enum directly, while in others we use the variable `ismType` might be better to stick one style no?